libraries/src/Installer/Adapter/ModuleAdapter.php

```diff
@@ -359,15 +359,23 @@ public function loadLanguage($path = null)
             $extension = $this->getElement();
 
             if ($extension) {
-                $source = $path ?: ($this->parent->extension->client_id ? JPATH_ADMINISTRATOR : JPATH_SITE) . '/modules/' . $extension;
+                $clientPath = ($this->parent->extension->client_id ? JPATH_ADMINISTRATOR : JPATH_SITE);
+                $source = $path ?: $clientPath . '/modules/' . $extension;
                 $folder = (string) $this->getManifest()->files->attributes()->folder;
 
                 if ($folder && file_exists($path . '/' . $folder)) {
                     $source = $path . '/' . $folder;
                 }
 
                 $client = (string) $this->getManifest()->attributes()->client;
-                $this->doLoadLanguage($extension, $source, \constant('JPATH_' . strtoupper($client)));
+                if (strlen($client))
+                {
+                    $this->doLoadLanguage($extension, $source, \constant('JPATH_' . strtoupper($client)));
+                }
+                else
+                {
+                    $this->doLoadLanguage($extension, $source, $clientPath);
+                }
```

Comment:
```suggestion
                if (strlen($client)) {
                    $this->doLoadLanguage($extension, $source, \constant('JPATH_' . strtoupper($client)));
                } else {
                    $this->doLoadLanguage($extension, $source, $clientPath);
                }
```
Fix remaining code style errors reported by drone.

Reply:
Brackets moved.

Reply:
My suggestion moves also the else up so it would fix the last code style error reported by drone: https://ci.joomla.org/joomla/joomla-cms/58340/1/8

Reply:
I really don’t wanna bug you and am happy about your PR. It’s just the other automated tests will not run as long as the code style check fails.

Reply:
else moved up.
Thanks for bugging me I need to get to grips with reliably and consistently submitting PRs.